ably-ios/ARTAuthTokenParams.m

```diff
@@ -130,7 +129,7 @@ - (NSDictionary *)toDictionaryWithUnion:(NSArray *)items {
     
     CCHmac(kCCHmacAlgSHA256, cKey, keyLen, cData, dataLen, hmac);
     NSData *mac = [[NSData alloc] initWithBytes:hmac length:sizeof(hmac)];
-    NSString *str = [ARTBase64PayloadEncoder toBase64:mac];
+    NSString *str = [mac base64EncodedStringWithOptions:0];
```

Comment:
Sorry, I thought it was something else. Even so, can you use the `enum NSDataBase64Encoding64CharacterLineLength` instead of `0`?